src/main/java/org/folio/rest/support/EffectiveCallNumberComponentsUtil.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.rest.support;
+
+import org.apache.commons.lang.StringUtils;
+import org.folio.rest.jaxrs.model.EffectiveCallNumberComponents;
+import org.folio.rest.jaxrs.model.HoldingsRecord;
+import org.folio.rest.jaxrs.model.Item;
+
+/**
+ * Utility methods for Effective Call Number Components object.
+ */
+public final class EffectiveCallNumberComponentsUtil {
+
+  private EffectiveCallNumberComponentsUtil() {
+    throw new UnsupportedOperationException("Do not instantiate");
+  }
+
+  public static EffectiveCallNumberComponents buildComponents(HoldingsRecord holdings, Item item) {
+    return updateComponents(new EffectiveCallNumberComponents(), holdings, item);
+  }
+
+  public static EffectiveCallNumberComponents updateComponents(EffectiveCallNumberComponents components, HoldingsRecord holdings, Item item) {
```

Comment:
Please can this signature be formatted to fit within the line length

Reply:
@marcjohnson-kint Is there a style guide somewhere I can refer to?

Reply:
Yes, although it is not complete - https://dev.folio.org/guidelines/contributing/#coding-style

Reply:
The [Java code standard](https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-136091.html#262) which FOLIO chose, defines the line limit as 80 characters.